src/containers/AccountSettings/accountSettings.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Icon } from 'antd';
+
+import Logo from '../../components/LogoHeader';
+import SubHeading from '../../components/subHeading';
+import NavigationBar from '../../components/navigationBar';
+
+import './accountSettings.css';
+
+const accountSettings = ({ info }) => {
+  const logOut = () => {
```

Comment:
Maybe this could be called handleLogOut and it should be moved to the index, I think